.github/workflows/ci.yaml

```diff
@@ -23,9 +23,11 @@ jobs:
         run: |
           go test ./internal/...
       - name: "Run Homerunner tests" # use a simple static dendrite image to sanity check homerunner works
+        env:
+          DOCKER_BUILDKIT: 1
         run: |
           mkdir -p homeserver
-          wget -O - "https://github.com/matrix-org/dendrite/archive/v0.8.8.tar.gz" | tar -xz --strip-components=1 -C homeserver
+          wget -O - "https://github.com/matrix-org/dendrite/archive/0489d16f95a3d9f1f5bc532e2060bd2482d7b156.tar.gz" | tar -xz --strip-components=1 -C homeserver
```

Comment:
I would have used the latest tag release myself. Is there any reason for this commit or is it just the latest on their main branch? (Can you leave a comment with the answer?)

Reply:
it's the latest, but last release doesn't cut it, it's still on stretch. Adding a comment.